ddl/placement_sql_test.go

```diff
@@ -454,6 +454,7 @@ PARTITION BY RANGE (c) (
 		tk.MustExec(fmt.Sprintf("set @@txn_scope = %v", testcase.txnScope))
 		if testcase.disableAutoCommit {
 			tk.MustExec("set @@autocommit = 0")
+			tk.MustExec("begin")
```

Comment:
Why do you add this line?

Reply:
It's fine to omit `begin`. If `autocommit=0`, one transaction will start automatically when you execute a SQL.

Reply:
But it's also fine to add it.